app/services/save_practice_service.rb

```diff
@@ -112,9 +112,20 @@ def update_department_practices
     practice_depts = @practice.department_practices
     if dept_params.present?
       dept_keys = dept_params.keys
+      dept_ids = Array.new
+      practice_depts.each do |v|
+        dept_ids.push(v[:department_id].to_s)
+      end
       dept_keys.each do |key|
         value = dept_params[key][:value]
         destroy = dept_params[key][:_destroy] === 'true'
+        if destroy === false
+          if dept_ids.include? value
```

Comment:
can we just do
```
dept_ids.push(value) unless dept_ids.include? value
```
?

Reply:
no need for if/else

Reply:
yes - to the one liner.  :)  good call.

Reply:
Actually.. if the dept_ids.include? the value... I want the code to go back to the top of the loop.. not continue through the loop...  I don't think the one liner will do that... hence the if... else.. with "next".
if dept_ids.include? value
            next
          else
            dept_ids.push(value)
          end

Reply:
oh, ok.
then it would be 
```
next if dept_ids.include? value
dept_ids.push(value)
```
either works for me